CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Bug Fixes:
 - Select the correct VS toolset for Ninja generators with CMake Presets. [#2423](https://github.com/microsoft/vscode-cmake-tools/issues/2423)
 - Fix unhandled exception with CMakePresets.json. [#2117](https://github.com/microsoft/vscode-cmake-tools/issues/2117)
 - Fix issues with compiler argument quoting when configuring IntelliSense. [#2563](https://github.com/microsoft/vscode-cmake-tools/pull/2563)
+- Add support for cpptools API V6. [2508](https://github.com/microsoft/vscode-cmake-tools/issues/2508)
```

Comment:
Is this a "bug fix" or an "improvement"?  This improvement fixes some bugs.  If you want to list it as a bug fix, we should state what bugs are fixed instead of that this adds v6 support (which our customers don't know anything about).  You can say that this fixes problems with shell quoting of compiler arguments.